tests/ert_tests/data/test_loader.py

```diff
@@ -285,9 +286,7 @@ def test_different_data_key(facade):
 
 
 @deprecation.fail_if_not_removed
-@pytest.mark.parametrize(
-    "func", [loader.load_general_data, loader.load_summary_data, loader.load_block_data]
-)
+@pytest.mark.parametrize("func", [loader.load_general_data, loader.load_summary_data])
 def test_deprecated_entry_points(facade, monkeypatch, func):
```

Comment:
Think you are removing the deprecated entry points, so this whole test can be deleted too?

Reply:
Thanks, fixed.